src/t8_forest.h

```diff
@@ -825,20 +826,56 @@ void                t8_forest_save (t8_forest_t forest);
- * function values), please see the functions in \ref t8_forest_vtk.h.
- */
-void                t8_forest_write_vtk (t8_forest_t forest,
-                                         const char *filename);
+ * \param [in]      forest              The forest to write.
+ * \param [in]      fileprefix          The prefix of the files where the vtk will
+ *                                      be stored. The master file is then fileprefix.pvtu
+ *                                      and the process with rank r writes in the file
+ *                                      fileprefix_r.vtu.
+ * \param [in]      write_treeid        If true, the global tree id is written for each element.
+ * \param [in]      write_mpirank       If true, the mpirank is written for each element.
+ * \param [in]      write_level         If true, the refinement level is written for each element.
+ * \param [in]      write_element_id    If true, the global element id is written for each element.
+ * \param [in]      write_ghosts        If true, each process additionally writes its ghost elements.
+ *                                      For ghost element the treeid is -1.
+ * \param [in]      curved_flag         If true, write the elements as curved element types from vtk.
+ * \param [in]      do_not_use_API      Do not use the VTK API, even if linked and available.
+ * \param [in]      num_data            Number of user defined double valued data fields to write.
+ * \param [in]      data                Array of t8_vtk_data_field_t of length \a num_data
+ *                                      providing the used defined per element data.
+ *                                      If scalar and vector fields are used, all scalar fields
+ *                                      must come first in the array.
+ * \return  True if successful, false if not (process local).
+ */
+int                 t8_forest_write_vtk_ext (t8_forest_t forest,
+                                             const char *fileprefix,
+                                             int write_treeid,
+                                             int write_mpirank,
+                                             int write_level,
+                                             int write_element_id,
+                                             int write_ghosts,
+                                             int curved_flag,
+                                             int do_not_use_API,
+                                             int num_data,
+                                             t8_vtk_data_field_t * data);
+
+/** Write the forest in a parallel vtu format. There is one master
+ * .pvtu file and each process writes in its own .vtu file.
+ * If linked, the VTK API is used.
+ * Forest must be committed when calling this function.
+ * This function is collective and must be called on each process.
+ * For more options use t8_forest_write_vtk_ext
```

Comment:
```suggestion
 * For more options use \ref t8_forest_write_vtk_ext
```

Reply:
The doxygen command `\ref` will create a link to the other function in the docu.